src/oc_erchef/apps/chef_objects/src/chef_s3_ops.erl

```diff
@@ -118,6 +118,8 @@ delete_file(OrgId, AwsConfig, Bucket, Checksum) ->
             {error, Checksum}
     end.
 
+%-include_lib("erlcloud/include/erlcloud_aws.hrl").
+
```

Comment:
could we remove this if it is not needed

Reply:
Done.